library/general/src/lib/installation/proposal_client.rb

```diff
@@ -270,5 +270,9 @@ def ask_user(_attrs)
     def description
       raise NotImplementedError, "Calling abstract method 'description'"
     end
+
+    def write
+      # doing nothing is OK. or is it? clarify the API, the docs, actual usage!
+    end
```

Comment:
I worry it is not OK. If you really call it with write, then you expect it do something.